lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -44,7 +46,7 @@
                 },
 
                 render: function () {
-                    this.$el.html(_.template(teamsTemplate));
+                    HtmlUtils.setHtml(this.$el, HtmlUtils.HTML(teamsTemplate));
```

Comment:
This doesn't seem right. You will be rendering the template itself into the page. I think you need to render the template into HTML, e.g.

    HtmlUtils.setHtml(this.$el, HtmlUtils.template(teamsTemplate)());